js/util/templateHelpers.js

```diff
@@ -36,6 +36,91 @@ define(["globals", "lib/faces", "lib/knockout", "util/helpers"], function (g, fa
         }
     };
 
+    ko.bindingHandlers.letterGrade = {
+      update: function(element, valueAccessor) {
+        var arg, output;
+        arg = ko.unwrap(valueAccessor());
+
+        if (arg < 0.20) {
```

Comment:
They may not, that just seemed like the most vanilla translation between the 0-1 scale and a letter scale.  I figured moving forward we would either monitor them to see if they pass the sniff test, or implement something like you had mentioned before with the (can't remember what it was called, maybe zRating?  Basically how many standard deviations it is away from basing it off of the league average)   If we do something like that though, would we also want to do something to normalize the displayed base ratings?